### DISCUSSION TITLE
Nuxt-config.ts is Different between "Getting Started" and "Menus, Data-drive UI, Styling"

### DISCUSSION THREAD

**Discussion by wadewa0352** [robconery/nuxt-walkthrough#4] [General]
I noticed that code I had for the nuxt-config was not the same as yours when I finished the "Getting Started" video.  Here is what I had:

```
export default defineNuxtConfig({
  css: ['vuetify/lib/styles/main.sass'],
  build: {
    transpile: ['vuetify'],
  },
  vite: {
    define: {
      'process.env.DEBUG': false
    }
  }
})
```

Not a big deal as I just made it match yours, but not sure if the changes are something that I missed or was missed in the video.  Figured I would mention it if it was the latter.  Thanks.

Wade

**Comment by robconery** [comment]
Thanks - what was different? I think the biggest change is `css` and `build` - don't think the `vite` setting is needed. Is that what's missing?

**Comment by wadewa0352** [comment]
@robconery it just seems that while I was watching "Menus, Data-drive UI, Styling" there was a big jump from where the "Getting Started" video ended.  Though the video seems different when I just watched it, so hopefully this resolved my issue.  Thanks.
